src/js/components.js

```diff
@@ -1,106 +1,106 @@
-  todo.id
-}" data-bs-toggle="modal" data-bs-target="#editTaskModal"><i class="bi bi-pencil-fill"></i></button>
-      </div>
-    </div>`;
-}
-
-export function makeTodoForm(todo = {}) {
-  const todoForm = parser(`
-    <form id="todoForm">
-      <div class="mb-3">
-        <label for="title" class="form-label">Title</label>
-        <input type="text" value="${
-  todo.title || ''
-}" class="form-control" id="title" name="title" placeholder="What is the title of your task?" required>
-      </div>
-      <div class="mb-3">
-        <label for="description" class="form-label">Description</label>
-        <textarea class="form-control" id="description" name="description" rows="3" placeholder="What is your task about?" value"${
-  todo.description || ''
-}" required></textarea>
-      </div>
-      <div class="mb-3">
-        <label for="date" class="form-label">Date</label>
-        <input class="form-control" type="date" id="date" name="date" required>
-      </div>
-      <div class="mb-3">
-        <label for="priority" class="form-label">Priotity</label>
-        <select class="form-select" id="priority" name="priority" value"${
-  todo.priority || 'low'
-}">
-            <option value="low">Low</option>
-            <option value="mid" selected>Mid</option>
-            <option value="high">High</option>
-        </select>
-      </div>
-    </form>
-  `);
-
-  return todoForm;
-}
-
-export function editTodoForm(todo = {}) {
-  const todoForm = `
-    <form id="editTodoForm" data-todoId=${todo.id}>
-      <div class="mb-3">
-        <label for="title" class="form-label">Title</label>
-        <input type="text" value="${
-  todo.title || ''
-}" class="form-control" id="title" name="title" placeholder="What is the title of your task?" required>
-      </div>
-      <div class="mb-3">
-        <label for="description" class="form-label">Description</label>
-        <textarea class="form-control" id="description" name="description" rows="3" placeholder="What is your task about?" required> ${
-  todo.description
-}
-        </textarea>
-      </div>
-      <div class="mb-3">
-        <label for="date" class="form-label">Date</label>
-        <input class="form-control" type="date" id="date" name="date" value="${format(
-    todo.date,
-    'yyyy-MM-dd',
-  )}" required>
-      </div>
-      <div class="mb-3">
-        <label for="priority" class="form-label">Priority</label>
-        <select class="form-select" id="priority" name="priority">
-            <option value="low"${
-  todo.priority === 'mid' && 'selected'
-}>Low</option>
-            <option value="mid" ${
-  todo.priority === 'mid' && 'selected'
-}>Mid</option>
-            <option value="high" ${
-  todo.priority === 'mid' && 'selected'
-}>High</option>
-        </select>
-      </div>
-    </form>
-  `;
-  return todoForm;
-}
+import { format } from 'date-fns';
+import { parser } from './utils';
+
+export function makeTodoCard(todo, index = null) {

```

Comment:
- [ ] You have to test the functions in component.js if they return the right component with the right class or not.
- for example
```
expect(YOUR_NODE_NAME.nodeName === 'SECTION' && YOUR_NODE_NAME.getAttribute('class')).toBe('col-12 col-md-5 d-flex text-dark h-100vh second-section');
```